stl/inc/shared_mutex

```diff
@@ -29,10 +29,7 @@ _EXPORT_STD class shared_mutex { // class for mutual exclusion shared across thr
 public:
     using native_handle_type = _Smtx_t*;
 
-    shared_mutex() noexcept // strengthened
-        : _Myhandle(nullptr) {}
-
-    ~shared_mutex() noexcept {}
+    shared_mutex() = default;
 

```

Comment:
Since we're touching this class, could we move the deleted special member function definitions from lines 62-63 up here after the default constructor where the Standard places them in the class synopsis?